### ISSUE TITLE
Handle noscript Better

### ISSUE THREAD

**Issue by rameshvarun** [rameshvarun/blog-cells#2]
Right now, with Javascript disabled, the code simply doesn't show at all. We should handle this better. Maybe let people create code cells in `<pre>` tags and transform them in-place into cells?

**Comment by Scepheo** [comment]
Alternatively, you could add CSS to make the `script` tags visible:

```css
script[type="text/notebook-cell"] {
  display: block;
}
```

This would also stop users from having to HTML-escape the content of `pre` tags.

**Comment by rameshvarun** [comment]
I had no idea this was possible - it works pretty well. You can even style with `white-space: pre`.

The one annoyance is that actual `pre` elements have some special logic to ignore the first line, as seen in the following HTML. Not really a big deal though, so I want to move forward with this approach.

```
<script type="text/notebook-cell">
console.log("Hello World!");
</script>

<pre class="notebook-cell">
console.log("Hello World!");
</pre>
```

<img width="734" alt="Screenshot 2023-08-29 at 10 00 41 PM" src="https://github.com/rameshvarun/blog-cells/assets/889879/d41efe6c-f3b1-4eb0-91e5-16068b018b51">

**Comment by rameshvarun** [comment]
Released in v0.5.0
